src/core/Akka.Remote.TestKit/CommandLine.cs

```diff
@@ -26,17 +26,19 @@ namespace Akka.Remote.TestKit
     ///     var testName = CommandLine.GetProperty("multinode.test-method");
     /// </code>
     /// </summary>
-    public class CommandLine
+    public static class CommandLine
     {
-        private static readonly StringDictionary Values;
+        private static readonly StringDictionary Values = new StringDictionary();
 
         static CommandLine()
         {
-            Values = new StringDictionary();
+            Initialize(Environment.GetCommandLineArgs());
```

Comment:
Ahhh got it - need to defer the `Environment.GetCommandLineArgs` call until we're ready...